### DISCUSSION TITLE
[deliver] Any way to automatically confirm the MIIT dialog?

### DISCUSSION THREAD

**Discussion by darkThanBlack** [fastlane/fastlane#22210] [Q&A]
Most mainland China users register their company names in Apple in English, but ICP filing requires Chinese. This will cause Apple to give a strong prompt as shown in the figure every time a new version is submitted:

![image-20240903172203416](https://raw.githubusercontent.com/darkThanBlack/darkThanBlack.github.io/pictures/docs/assets/pictures/image-20240903172203416.png)

We need to manually confirm it:
![image-20240903172258447](https://raw.githubusercontent.com/darkThanBlack/darkThanBlack.github.io/pictures/docs/assets/pictures/image-20240903172258447.png)

This is fine, but it will cause automatic submission review errors:
![image-20240903174008632](https://raw.githubusercontent.com/darkThanBlack/darkThanBlack.github.io/pictures/docs/assets/pictures/image-20240903174008632.png)

Although the ipa file will still upload successfully, is there anyway to fix it with script, or improve my `deliver` pipline?

**Comment by Norcy** [comment]
+1
